src/components/dialogs/input-hooks.js

```diff
@@ -397,13 +399,39 @@ export const useConnectivityValue = ({
 
 const filter = createFilterOptions();
 
+const QUESTIONABLE_SIZE = 1000;
```

Comment:
change constant name ?

Reply:
May be, but change it to what ? I came using this name, because it was the size for which it is better to ask again the new term having previous one as prefix.